<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportDvd</class>
 <widget class="QDialog" name="DialogImportDvd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>اضافة كتب قرص مضغوط</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>حدد المجلد الذي يحتوي على الكتب المضغوطة سواء في القرص المضغوط او في اي قسم تم نسخها اليه</string>
     </property>
     <property name="text">
      <string>حدد المجلد الذي يحتوي على الكتب المضغوطة </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEditDvdPath">
       <property name="toolTip">
        <string>حدد المجلد الذي يحتوي على الكتب المضغوطة سواء في القرص المضغوط او في اي قسم تم نسخها اليه</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="toolTip">
        <string>قم بتحديد المجلد الذي يحتوي على الكتب المضغوطة سواء في القرص المضغوط او في اي قسم قمت بنسخها اليه</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelBooksPath">
           <property name="toolTip">
            <string>حدد مسار الكتب التذي سيتم ازالة الضغط فيه</string>
           </property>
           <property name="text">
            <string>مسار الكتب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditBooksPath"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonBooksPath">
           <property name="toolTip">
            <string>حدد مسار الكتب التذي سيتم ازالة الضغط فيه</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxGroup">
         <property name="text">
          <string>نسخ قائمة الكتب الجديدة</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>حدد المجموعات او الكتب التي تريد ازالة الضغط عنها واضافتها الى مسار الكتب</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string>المجموعة</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ملاحظة : تحديد مجموعة مجموعة  افضل واسرع من تحديد كتاب كتاب</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEditConsole"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>الرجاء الانتظار ريثما ينتهي المعالج من استخراج الملفات</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogImportDvd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
